modules/csg/csg.cpp

```diff
@@ -37,16 +37,16 @@
 // Static helper functions.
 
 inline static bool is_snapable(const Vector3 &p_point1, const Vector3 &p_point2, real_t p_distance) {
-	return (p_point1 - p_point2).length_squared() < p_distance * p_distance;
+	return p_point2.distance_squared_to(p_point1) < p_distance * p_distance;
 }
 
 inline static Vector2 interpolate_segment_uv(const Vector2 p_segment_points[2], const Vector2 p_uvs[2], const Vector2 &p_interpolation_point) {
-	float segment_length = (p_segment_points[1] - p_segment_points[0]).length();
+	float segment_length = p_segment_points[0].distance_to(p_segment_points[1]);
 	if (p_segment_points[0].is_equal_approx(p_segment_points[1])) {
 		return p_uvs[0];
 	}
 
```

Comment:
This can be improved further:

```suggestion
	if (p_segment_points[0].is_equal_approx(p_segment_points[1])) {
		return p_uvs[0];
	}
	float segment_length = p_segment_points[0].distance_to(p_segment_points[1]);
```